### PR TITLE
OptCached is using Footprint (#1608)

### PR DESCRIPTION
**Pull request by PickBas**
Closes: #1608

### CODE REVIEW

**Comment by PickBas** [comment]
@mximp Please, have a look

File: eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptCached.java [hunk 1]
```diff
@@ -58,8 +58,7 @@ public final class OptCached implements Optimization {
      */
     public OptCached(
         final Optimization delegate,
-        final Path folder
-    ) {
+        final Path folder) {
```
Comment (mximp) [inline eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptCached.java:61]: @PickBas closing brace should start at the same indent as the statement with corresponding opening brace. So it was correct.

Comment (PickBas) [reply eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptCached.java]: @mximp Fixed

**Review by mximp — APPROVED** [review]

**Comment by mximp** [comment]
@yegor256 please merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**